src/Maestro/Maestro.ContainerApp/Queues/QueueProcessors/PullRequestReminderQueueProcessor.cs

```diff
@@ -40,5 +48,7 @@ protected override async Task ProcessAsyncInternal(PullRequestReminderWorkItem w
         {
             await pullRequestActor.ProcessPendingUpdatesAsync();
         }
+
+        await _database.StringGetDeleteAsync(workItem.Name);
```

Comment:
Line 40: `throw new ArgumentException(...)`
We can throw an exception that the actorid is invalid. do we need to ensure/to delete workItem from DB if the exception is thrown?

Reply:
good point

Reply:
I'd imagine the whole queue processing and actors would need to get hardened against exceptions and a retry mechanism defined. But that would be for production service

Reply:
But yeah, reminder deletion should go inside of `finally {}` of some sort

Reply:
for now I changed it to remove the message in the beginning